language/en-GB/mod_articles.ini

```
@@ -70,6 +70,7 @@ MOD_ARTICLES_FIELD_TITLE_HEADING_NONE="None"
 MOD_ARTICLES_FIELD_TITLE_LABEL="Article Title"
 MOD_ARTICLES_FIELD_TRIGGER_EVENTS_DESC="Enable this option to trigger three plugin events (afterDisplayTitle, beforeDisplayContent, afterDisplayContent) for each article in the results list."
 MOD_ARTICLES_FIELD_TRIGGER_EVENTS_LABEL="Trigger Plugin Events"
+MOD_ARTICLES_FIELD_UNPUBLISHED_LABEL="Show unpublished articles"
```

```suggestion
MOD_ARTICLES_FIELD_UNPUBLISHED_LABEL="Show Unpublished Articles"
```

please follow the style guide